packages/steps/src/__tests__/utils/context.ts

```diff
@@ -93,6 +95,7 @@ export function createGlobalContextMock({
       workingDirectory ?? resolvedProjectTargetDirectory,
       staticContextContent ?? {}
     ),
-    skipCleanup ?? false
+    skipCleanup ?? false,
+    configPath ?? './fake.yml'
```

Comment:
Should `fake.yml` be defined somewhere or something?

Reply:
We don't need it to exist for the purpose of this test